### PR TITLE
Add libopus AVX settings

### PR DESCRIPTION
**Pull request by zziger**
* Before adding new features and new modules, please go to issues to submit the relevant feature description first.
* Write good commit messages and use the same coding conventions as the rest of the project.
* Please commit code to dev branch and we will merge into master branch in feature
* Ensure your edited codes with four spaces instead of TAB.

------

* 增加新特性和新模块之前，请先到issues提交相关特性说明，经过讨论评估确认后，再进行相应的代码提交，避免做无用工作。
* 编写友好可读的提交信息，并使用与工程代码相同的代码规范，代码请用4个空格字符代替tab缩进。
* 请提交代码到dev分支，如果通过，我们会在特定时间合并到master分支上。
* 为了规范化提交日志的格式，commit消息，不要用中文，请用英文描述。

### CODE REVIEW

File: packages/l/libopus/xmake.lua [hunk 1]
```diff
@@ -9,13 +9,18 @@ package("libopus")
     add_versions("1.3.1", "65b58e1e25b2a114157014736a3d9dfeaad8d41be1c8179866f144a2fb44ff9d")
     add_patches("1.3.1", path.join(os.scriptdir(), "patches", "1.3.1", "cmake.patch"), "79fba5086d7747d0441f7f156b88e932b662e2d2ccd825279a5a396a2840d3a2")
 
+    add_configs("avxSupported", { description = "AVX supported", default = true, type = "boolean" })
```
Comment (waruqi) [inline packages/l/libopus/xmake.lua:12]: please use lower case. rename to `avx`

File: packages/l/libopus/xmake.lua [hunk 2]
```diff
@@ -9,13 +9,18 @@ package("libopus")
...
+    add_configs("opusx86MayHaveAvx", { description = "Does runtime check for AVX support", default = true, type = "boolean" })
```
Comment (waruqi) [inline packages/l/libopus/xmake.lua:13]: rename to `check_avx `

File: packages/l/libopus/xmake.lua (hunk 1 above)
Comment (zziger) [reply packages/l/libopus/xmake.lua]: Done

File: packages/l/libopus/xmake.lua (hunk 2 above)
Comment (zziger) [reply packages/l/libopus/xmake.lua]: Done

**Pull request merged**